### PR TITLE
Update release notes for docker-compose 1.25.0

### PR DESCRIPTION
**Pull request by ulyssessouza**
### Proposed changes

Update release notes for docker-compose 1.25.0

### CODE REVIEW

File: release-notes/docker-compose.md [hunk 1]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+### Bugfixes
+
+- Make container service color deterministic, remove red from chosen colors
+
+- Fix non ascii chars error. Python2 only
+
+- Format image size as decimal to be align with Docker CLI
+
+- Use Python Posix support to get tty size
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: tty size acquired via Python Posix support.

File: release-notes/docker-compose.md [hunk 2]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fix same file 'extends' optimization
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Reword: Fixed same file 'extends' optimization.

File: release-notes/docker-compose.md [hunk 3]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Use python POSIX support to get tty size
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Delete. Repeat.

File: release-notes/docker-compose.md [hunk 4]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Format image size as decimal to be align with Docker CLI
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Delete. Repeat.

File: release-notes/docker-compose.md [hunk 5]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed stdin_open
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Add period to end of sentence.

File: release-notes/docker-compose.md [hunk 6]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed `--remove-orphans` when used with `up --no-start`
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: What was the problem? The sentence would be better stated as follows;

Fixed the `--remove-orphans` issue encountered during use with `up --no-start` option ({{between these paranethesis, a phrase describing the issue}}).

File: release-notes/docker-compose.md [hunk 7]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed `docker-compose ps --all`
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Add the word 'command' to the end, and a period.

File: release-notes/docker-compose.md [hunk 8]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed `depends_on` dependency recreation behavior
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Put the word 'the' after 'Fixed' and end the sentence with a period.

File: release-notes/docker-compose.md [hunk 9]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed bash completion for `build --memory`
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Reword: Fixed bash completion for the `build --memory` command.

File: release-notes/docker-compose.md [hunk 10]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed misleading warning concerning env vars when performing an `exec` command
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Reword: Fixed the misleading environmental variables warning that occurs when the `exec` command is performed.

File: release-notes/docker-compose.md [hunk 11]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed failure check in parallel_execute_watch
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Reword: Fixed the failure check in the parallel_execute_watch function.

File: release-notes/docker-compose.md [hunk 12]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed race condition after pulling image
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Reword: Fixed the race condition that occurs following the pulling of an image.

File: release-notes/docker-compose.md [hunk 13]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed error on duplicate mount points
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Could use a phrase about the error...what type of error?

File: release-notes/docker-compose.md [hunk 14]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed merge on networks section
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Is the section "merge on networks"? Or is it a merge that was fixed on a "networks" section? Not clear, and more detail could help.

File: release-notes/docker-compose.md [hunk 15]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Always connect Compose container to `stdin`
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Compose container is always connected to `stdin`.

File: release-notes/docker-compose.md [hunk 16]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Fixed the presentation of failed services on 'docker-compose start' when containers are not available
```
Comment (KoryKessel-Docker) [inline release-notes/docker-compose.md]: Change:  'docker-compose start' --> the docker-compose start command, and add a period to the end.

File: release-notes/docker-compose.md [hunk 17]
```diff
@@ -5,6 +5,103 @@ keywords: release notes, compose
...
+
+- Use improved API fields for project events when possible
+
+- Update `setup.py` for modern `pypi/setuptools` and remove `pandoc` dependencies
+
+- Removed `Dockerfile.armhf` which is no longer needed
+
+### Bugfixes
+
+- Make container service color deterministic, remove red from chosen colors
```
[the remark this answers is not in the document]
Comment (ulyssessouza) [reply release-notes/docker-compose.md]: That means that the first container will always have color A, the second, color B and so on...

File: release-notes/docker-compose.md (hunk 6 above)
Comment (ulyssessouza) [reply release-notes/docker-compose.md]: Replaced with
```
- Fixed the issue of `--remove-orphans` being ignored encountered during use with `up --no-start` option.
```

File: release-notes/docker-compose.md (hunk 14 above)
Comment (ulyssessouza) [reply release-notes/docker-compose.md]: yep. It's just that the merge in between 2 files was not good in a certain section (network). Replaced by:

```
- Fixed the merge on `networks` section.
```

**Pull request merged**